src/.clang-format

```diff
@@ -25,53 +25,74 @@ IndentWidth: 4
 UseTab: Never
 TabWidth: 8
 AccessModifierOffset: -4
-ConstructorInitializerIndentWidth: 4
-ContinuationIndentWidth: 4
+ConstructorInitializerIndentWidth: 0
```

Comment:
The exact problem was that a PR that changing how the XML files are read in has nothing to do with changing .clang-format settings - a separate PR with the right scope would have been fine but was never attempted.